tools/mo/openvino/tools/mo/main.py

```diff
@@ -373,13 +373,6 @@ def check_fallback(argv: argparse.Namespace):
 
 
 def prepare_ir(argv: argparse.Namespace):
```

Comment:
why is this deleted?

Reply:
Will be returned in the next PR along with tests.